love/src/Utils.js

```diff
@@ -2102,6 +2102,17 @@ export function getObsDayFromDate(date) {
   return utcDate.subtract(1, 'day').format(ISO_INTEGER_DATE_FORMAT);
 }
 
+/**
+ * Converts the given OBS day (YYYYMMDD) to ISO format (YYYY-MM-DD).
```

Comment:
Please use an active voice in doc strings. In this case `Converts` -> `Convert`.